huaweicloud/services/secmaster/resource_huaweicloud_secmaster_incident.go

```diff
@@ -0,0 +1,589 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product SecMaster
+// ---------------------------------------------------------------
+
```

Comment:
delete the file head

Reply:
This is needed when analyzing the APIs used with tool